### ISSUE TITLE
`comment-is-too-wide` incorrectly reports over-wide lines at the start of multi-line comments

### ISSUE THREAD

**Issue by gemshrine** [objectionary/lints#1353]
`comment-is-too-wide` splits a multi-line comment into its lines and reports each over-wide line at the comment's **first** line, so the number in the defect points at a line that is very likely fine.

`src/main/resources/org/eolang/lints/comments/comment-is-too-wide.xsl`:

```xml
<xsl:variable name="line" select="if (@line) then @line else '0'"/>
<xsl:variable name="lines" select="tokenize(replace(., '\n', '&#10;'), '&#10;')"/>
<xsl:choose>
  <xsl:when test="count($lines) &gt; 1">
    <xsl:for-each select="$lines[string-length(.) &gt; $max]">
      <xsl:element name="defect">
        <xsl:attribute name="line">
          <xsl:value-of select="$line"/>
        </xsl:attribute>
```

`$line` is read once, before the loop, from the comment element. Every defect the loop emits carries that same value. A comment whose fourth line is 120 characters wide is reported on the line the comment *starts* on, and a comment with three over-wide lines produces three defects that all name that one line.

The rule is the only one in this directory that walks inside a comment, so it is the only one where the comment's own line and the offending line differ. Its single-line branch, further down, is correct by construction — there the two are the same line.

The offset is available where the defect is built, but not from `position()`: the predicate `[string-length(.) > $max]` filters first, so inside that `for-each` the position is the index among the *offending* lines, not among all of them. Walking every line and testing inside the loop keeps the index meaningful:

```xml
<xsl:for-each select="$lines">
  <xsl:if test="string-length(.) &gt; $max">
    <xsl:element name="defect">
      <xsl:attribute name="line">
        <xsl:value-of select="number($line) + position() - 1"/>
      </xsl:attribute>
```

with `number($line)` guarding against the `'0'` fallback being concatenated rather than added.

Worth checking the `'0'` path in the same change. When the comment carries no `@line`, every defect gets line `0` and a `context` attribute, which is the intended fallback; with the offset added, `0 + position() - 1` would produce line numbers counting from zero for a comment whose real position is unknown. Keeping the fallback at `0` when `@line` is absent, and offsetting only when it is present, avoids inventing a location.

**Comment by 0crat** [comment]
@gemshrine Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+3542](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
